epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -86,6 +126,27 @@ private void cancelAnimation(LottieAnimationView lottieView) {
   }
 
   @Override
+  public boolean canAnimateChanges(ColorHolder view) {
+    return false;
+  }
+
+  @Override
+  public Animator animateChanges(ColorHolder view) {
```

Comment:
`animateChanges` and `canAnimateChanges` are from the previous example, I just haven't removed them yet.